### PR TITLE
refactor: Reformat code according to pep8 code style

### PR DESCRIPTION
**Pull request by elias-gvin**
1. Reformat code
2. Fix Effect class

### CODE REVIEW

**Review by SurprisedBird — APPROVED** [review]

File: effects/effect.py [hunk 1]
```diff
@@ -31,7 +26,7 @@ def __init__(self,
         self._activation_round: int
 
     def activate(self) -> None:
-        if self._status == EffectStatus.CREATED:
+        if (self._status == EffectStatus.CREATED):
```
Comment (asternotus) [inline effects/effect.py:29]: Are brackets necessary?

File: effects/effect.py [hunk 2]
```diff
@@ -60,13 +55,13 @@ def _validate(self, target_number: int) -> bool:
         pass
 
     def resolve(self) -> None:
-        if self._status == EffectStatus.ACTIVATED:
+        if (self._status == EffectStatus.ACTIVATED):
```
Comment (asternotus) [inline effects/effect.py:58]: Are brackets necessary?

File: effects/effect.py [hunk 3]
```diff
@@ -60,13 +55,13 @@ def _validate(self, target_number: int) -> bool:
...
             is_finished = self._resolve_impl()
 
             if (is_finished):
                 self._status = EffectStatus.FINISHED
 
-        else if self._status == EffectStatus.CREATED:
+        elif (self._status == EffectStatus.CREATED):
```
Comment (asternotus) [inline effects/effect.py:64]: Are brackets necessary?

File: effects/effect.py (hunk 1 above)
Comment (elias-gvin) [reply effects/effect.py:29]: Not necessary. Added for consistency with other code parts

File: effects/effect.py (hunk 2 above)
Comment (elias-gvin) [reply effects/effect.py:58]: Not necessary. Added for consistency with other code parts

File: effects/effect.py (hunk 3 above)
Comment (elias-gvin) [reply effects/effect.py:64]: Not necessary. Added for consistency with other code parts

**Review by asternotus — CHANGES_REQUESTED** [review]
Please change it o give an answer.

File: effects/effect.py (hunk 1 above)
Comment (asternotus) [reply effects/effect.py:29]: I always write without brackets. Now it is approved but we need to speak about one format for brackets, quotes etc. for each other.

File: effects/effect.py (hunk 2 above)
Comment (asternotus) [reply effects/effect.py:58]: I always write without brackets. Now it is approved but we need to speak about one format for brackets, quotes etc. for each other.

File: effects/effect.py (hunk 3 above)
Comment (asternotus) [reply effects/effect.py:64]: I always write without brackets. Now it is approved but we need to speak about one format for brackets, quotes etc. for each other.

**Review by asternotus — APPROVED** [review]
Now it is approved but we need to speak about one format for brackets, quotes etc. for each other.

**Pull request merged**